### ISSUE TITLE
Add `quiet` option

### ISSUE THREAD

**Issue by danlynkew** [levz0r/gmail-tester#108]
Package currently pollutes Cypress test logs with output such as below. @levz0r, please add a `quiet` option for us! 😄 And thank you for this wonderful package!

```[gmail] Checking for message from 'test@gmail.com', to: undefined, contains 'Verify your current email' in subject...
[gmail] Found!
[gmail] Checking for message from 'test@gmail.com', to: undefined, contains 'Verify your current email' in subject...
[gmail] Found!
[gmail] Checking for message from 'test@gmail.com', to: undefined, contains 'Change your email' in subject...
[gmail] Message not found. Waiting 1 seconds...
[gmail] Message not found. Waiting 1 seconds...
[gmail] Message not found. Waiting 1 seconds...
[gmail] Message not found. Waiting 1 seconds...
[gmail] Found!
[gmail] Checking for message from 'test@gmail.com', to: undefined, contains 'Change your email' in subject...
[gmail] Found!```

**Comment by tracy-ash** [comment]
Yes, please! Quiet option!

**Comment by danlynkew** [comment]
Thanks for the response, @levz0r. I tried calling `get_messages()` instead of `check_inbox()`. But `get_messages()` seems to instantly return me an empty array. I've confirmed that the messages are in the inbox, and they're always detected properly by `check_inbox()`.

I tried to follow your `get_messages()` example in the docs. Am I using `get_messages()` incorrectly?

<img width="643" alt="Screen Shot 2022-12-21 at 5 00 28 PM" src="https://user-images.githubusercontent.com/38993262/209002164-ae47dc3d-82e0-4fd7-9874-266b3171eab6.png">

<img width="385" alt="Screen Shot 2022-12-21 at 5 00 15 PM" src="https://user-images.githubusercontent.com/38993262/209002176-eb0777f5-da3f-4133-a4f8-cf99e21807ac.png">
